pages/k8s/quickstart.md

```diff
@@ -30,156 +31,133 @@ With this quick start guide and some tools from Canonical, you'll have a Kuberne
+          <div class="p-list-step__content">
+
+<a class="p-link--external" href="https://jaas.ai" > Juju</a> is a tool for
+deploying, configuring, and operating complex software on public or private
+clouds. It can be installed with a snap:
+
+            <div class="p-code-snippet">
+              <input class="p-code-snippet__input" value="sudo snap install juju --classic" readonly="readonly">
+              <button class="p-code-snippet__action">Copy to clipboard</button>
+            </div>
+            <script id="asciicast-254739" src="https://asciinema.org/a/254739.js" async data-autoplay="true" data-rows="4"></script>
+          </div>
+        </li>
+
+        <li class="p-list-step__item col-12">
+          <h3 class="p-list-step__title"><span class="p-list-step__bullet">2</span>Find your cloud</h3>
+          <div class="p-list-step__content">
+
+Juju has baked in knowledge of many public clouds such as AWS, Azure and
+Google. You can see which ones are ready to use by running this command:
+
+            <div class="p-code-snippet">
+              <input class="p-code-snippet__input" value="juju clouds" readonly="readonly">
+              <button class="p-code-snippet__action">Copy to clipboard</button>
+            </div>
+            <script id="asciicast-254740" src="https://asciinema.org/a/254740.js" async data-rows="18" ></script>
+
+            <p><a class="p-link--external" href="https://docs.jujucharms.com/clouds">Find out more about Clouds in Juju</a></p>
+          </div>
+        </li>
+
+        <li class="p-list-step__item col-12">
+          <h3 class="p-list-step__title"><span class="p-list-step__bullet">3</span>Add Credentials</h3>
+          <div class="p-list-step__content">
+<p>Most clouds require credentials so that the cloud knows which operations are authorised, so you will need to supply these for Juju. If you choose to use AWS, for example, you would run:</p>
+            <div class="p-code-snippet">
+              <input class="p-code-snippet__input" value="juju add-credential aws" readonly="readonly">
+              <button class="p-code-snippet__action">Copy to clipboard</button>
+            </div>
+<p>For a different cloud, just substitute in the name from the previous
+   list output by Juju. The data you need to supply will vary depending on the cloud. </p>
+             <script id="asciicast-Wo12W39et3IJzF15rAyVunbbl" src="https://asciinema.org/a/Wo12W39et3IJzF15rAyVunbbl.js" async data-rows="18" ></script>
+          </div>
+
+        </li>
+
+        <li class="p-list-step__item col-12">
+          <h3 class="p-list-step__title"><span class="p-list-step__bullet">4</span>Add a Controller</h3>
+          <div class="p-list-step__content">
+            <p>The Juju controller is used to manage the software deployed through Juju, from deployment to upgrades to day-two operations. One Juju controller can manage multiple projects.</p>
```

Comment:
Should we use juju nomenclature here? projects vs models?

Reply:
Hmm. I took this wording from the looked-at-by-everyone kubernetes pages on ubuntu.com
I don't have a firm opinion on it, but I would say that 'projects' is more easily grokable than 'models' to those unfamiliar with Juju. Maybe tim has an opinion.

Reply:
I get that and it is easier if the desire is to walk them in, but right below this, we talk about models. I'm not sure it is the best idea to use model before we define it though.